libsel4vm/arch_include/x86/sel4vm/arch/vmcs_fields.h

```diff
@@ -95,3 +95,23 @@
 #define VMX_CONTROL_EXIT_CONTROLS 0x0000400C
 #define VMX_CONTROL_ENTRY_INTERRUPTION_INFO 0x00004016
 #define VMX_CONTROL_ENTRY_EXCEPTION_ERROR_CODE 0x00004018
+
+/* Access rights bits */
+
+#define AR(x)   (ACCESS_RIGHTS_##x)
+#define AR_T(x) (ACCESS_RIGHTS_TYPE_##x)
+
+#define ACCESS_RIGHTS_S        BIT(4)
+#define ACCESS_RIGHTS_P        BIT(7)
+#define ACCESS_RIGHTS_AVL      BIT(12)
+#define ACCESS_RIGHTS_L        BIT(13)
+#define ACCESS_RIGHTS_DB       BIT(14)
+#define ACCESS_RIGHTS_G        BIT(15)
+#define ACCESS_RIGHTS_UNUSABLE BIT(16)
+
+#define ACCESS_RIGHTS_TYPE_BUSY_TSS    0xB
+#define ACCESS_RIGHTS_TYPE_BUSY_16_TSS 0x3
+#define ACCESS_RIGHTS_TYPE_LDT         0x2
+
+#define DEFAULT_TR_ACCESS_RIGHTS AR(P) | AR_T(BUSY_TSS)
```

Comment:
Please use brackets here and in the other macro to make the expression an atoms.